apps/blockchain/lib/blockchain/bloom.ex

```diff
@@ -1,4 +1,9 @@
 defmodule Blockchain.Bloom do
+  @moduledoc """
+  When a block is generated or verified, the contract addresses and fields from the generated logs are added to a bloom filter. This is included in the block header.
+
+  _From Yellow Paper 4.3.1. Transaction Receipt_: The transaction receipt is a tuple of four items comprising the post-transaction state, R, the cumulative gas used in the block containing the transaction receipt as of immediately after the transaction has happened, Ru, the set of logs created through execution of the transaction, Rl and the Bloom filter composed from information in those logs, Rb:
```

Comment:
What do you think about breaking this paragraph (and the previous one) into multiple lines? It makes it a bit hard to read like this.

Reply:
I also noticed that this ends with `Rb:`. Is there supposed to be something after that or is the colon supposed to be a period?